source/clear-linux/get-started/get-started.rst

```diff
@@ -0,0 +1,23 @@
+.. _get-started:
+
+Get started
+###########
+
+This section is where you will find documentation to do with the
```

Comment:
Wrong tone and geography. Consider instead: This section contains information about the installation of |CLOSIA|.